indy-besu/smart_contracts/contracts/did/DidRegistry.sol

```diff
@@ -48,14 +57,17 @@ contract DidRegistry is DidRegistryInterface, ControlledUpgradeable {
         DidValidator.validateVerificationKey(document);
 
         _dids[document.id].document = document;
+        _dids[document.id].metadata.creatorAddress = msg.sender;
```

Comment:
Can it be just `creator` to have consistent naming with other contracts?

Reply:
Renamed